src/Components/Components/src/ComponentBase.cs

```diff
@@ -332,5 +334,17 @@ Task IHandleAfterRender.OnAfterRenderAsync()
             // have to use "async void" and do their own exception handling in
             // the case where they want to start an async task.
         }
+
+        private string DebuggerDisplayString()
+        {
+            if (!_renderHandle.IsInitialized)
+            {
+                return "(uninitialized) " + ToString();
+            }
+            else
+            {
+                return $"({_renderHandle.ComponentId}) " + ToString();
```

Comment:
Do you need the `ToString()` calls?

Reply:
Yes, because that's the default VS does when there is no attribute, and you want to see the component type